### PR TITLE
Milestone 3: Using API

### PR DESCRIPTION
**Pull request by eroiyo**
Hello Microverse Community, this is the milestone where we add the API functions inside the page, now we do fetch requests using axios, that maybe is a bit different from thunk, but have the same effects

*Delete books from the API
*Add books from the API
*Show books from the API

It says "Coming soon" in the author section because the API dont have the capacity to save authors name, so i will add it later when i am able to use other API.

### CODE REVIEW

File: src/Api.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+/* eslint-disable linebreak-style */
+import axios from 'axios';
+
```
Comment (Emmyn5600) [inline src/Api.js:3]: - [ ] Kindly fix the eslint errors correctly without disable it.

File: src/Api.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    method: 'DELETE',
+    url: newURL,
+    data: {
+      item_id: id,
+    },
+  }).catch((err) => {
+    console.log('Error', err);
+  });
+  return (response.data);
+};
```
Comment (Emmyn5600) [inline src/Api.js:44]: - [ ] Let us remove the console.log in the code you have pushed.

File: src/redux/books/Book.js [hunk 3]
```diff
@@ -1,7 +1,9 @@
 /* eslint-disable linebreak-style */
+
```
Comment (Emmyn5600) [inline src/redux/books/Book.js:2]: - [ ] Kindly fix the eslint errors correctly without disable it.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @eroiyo  

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:. 
   
- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Changes required :writing_hand: 

      * #### Please check the comments under the review.
       
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by eroiyo** [comment]
Thanks, i will work in that

**Pull request merged**